commit cb459d7e301c28f3ba12440aa8fbe8b26747cd04 (refs/changes/55/2855/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-24T22:09:51-07:00 (1 year, 11 months ago)
    
    TOOLS-1898 sdcnode updates for Node.js Oct 2017 vulns (add sdcnode v4.8.5 builds)

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 016a7f4..fc12318 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -1,4 +1,31 @@
 [
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "version": "v4.8.5",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "gcc_version_prefix": "4.9.",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v4.8.5",
+        "configure_flags": "--without-snapshot --dest-cpu=ia32 --with-dtrace",
+        "build_tag": "zone"
+    },
+    {
+        "// image": "sdc-minimal-multiarch-lts 15.4.1",
+        "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v4.8.5",
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "build_tag": "zone64"
+    },
+
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
